tm2/pkg/sdk/params/handler_test.go

```diff
@@ -31,11 +31,11 @@ func TestQuery(t *testing.T) {
 		path     string
 		expected string
 	}{
-		{path: "params/params_test/foo/bar.string", expected: `"baz"`},
-		{path: "params/params_test/foo/bar.int64", expected: `"-12345"`},
-		{path: "params/params_test/foo/bar.uint64", expected: `"4242"`},
-		{path: "params/params_test/foo/bar.bool", expected: "true"},
-		{path: "params/params_test/foo/bar.bytes", expected: `"YmF6"`},
+		{path: "params/foo/bar.string", expected: `"baz"`},
```

Comment:
these dots are misleading (compared to previous revisions of the code), and the gno.land toml syntax for params.
maybe replace with underscore.